lib/cart/widgets/cart_info_card.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flex_storefront/flex_ui/tokens/colors.dart';
+import 'package:flex_storefront/flex_ui/tokens/sizes.dart';
+import 'package:flutter/material.dart';
+
+enum CartInfoType {
+  info,
+  success,
+  warning,
+  error,
+}
+
+class CartInfoCard extends StatelessWidget {
```

Comment:
A bit strange to have a CartInfoCard that can have another type than "info".

How about CartFeedback or something?

Reply:
Oh nevermind, it's more generic than that